skynet/env.py

```diff
@@ -26,16 +26,16 @@ def tobool(val: str | None):
 
 # models
 llama_path = os.environ.get('LLAMA_PATH')
-llama_n_ctx = int(os.environ.get('LLAMA_N_CTX', 8192))
-llama_n_gpu_layers = int(os.environ.get('LLAMA_N_GPU_LAYERS', -1 if is_mac else 40))
+llama_n_ctx = int(os.environ.get('LLAMA_N_CTX', 32000))
```

Comment:
Is this as large as we can go on the A10? Just a thought: the default should be what the model can do, and we can tweak it as we deploy it.